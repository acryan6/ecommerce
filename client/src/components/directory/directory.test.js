import { shallow } from "enzyme";
import React from "react";
import { Directory } from "./directory.component.jsx";

it("expect to render directory component", () => {
  expect(shallow(<Directory sections={[]} />)).toMatchSnapshot();
});
