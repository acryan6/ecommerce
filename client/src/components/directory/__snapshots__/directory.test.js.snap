// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render directory component 1`] = `ShallowWrapper {}`;
